packages/popup/src/pages/ImportAccount/index.js

```diff
@@ -0,0 +1,166 @@
+import {useState, useEffect} from 'react'
+import {useParams, useHistory} from 'react-router-dom'
+import {useTranslation} from 'react-i18next'
+import {TextNav} from '../../components/index'
+import Button from '@cfxjs/component-button'
+import Input from '@cfxjs/component-input'
+import {CompWithLabel} from '../../components'
+import {useRPC} from '@cfxjs/use-rpc'
+import {request} from '../../utils'
+import {
+  GET_HD_ACCOUNT_GROUP,
+  GET_ALL_ACCOUNT_GROUP,
+  GET_PK_ACCOUNT_GROUP,
+} from '../../constants'
+import useGlobalStore from '../../stores'
+import {useSWRConfig} from 'swr'
+
+function ImportAccount() {
+  const {t} = useTranslation()
+  const {pattern} = useParams()
+  const history = useHistory()
+  const {mutate} = useSWRConfig()
+
+  const [name, setName] = useState('')
+  const [keygen, setKeygen] = useState('')
+  const [keygenErrorMessage, setKeygenErrorMessage] = useState('')
+  const [keygenNamePlaceholder, setKeygenNamePlaceholder] = useState('')
+  const [creatingAccount, setCreatingAccount] = useState(false)
+  const createdPassword = useGlobalStore(state => state.createdPassword)
+
+  let lanPrefix = ''
+  let deps = null
+  let groupParams = undefined
```

Comment:
better seperate these two pages now, there will be major difference in the futuree

Reply:
done